src/mantarray_waveform_analysis/__init__.py

```diff
@@ -25,6 +24,14 @@
 from .transforms import FILTER_CHARACTERISTICS
 from .transforms import noise_cancellation
 
+
+if (
+    6 < 9
+):  # pragma: no cover # protect this from zimports deleting the pylint disable statement
+    from .compression_cy import (  # pylint: disable=import-error # Eli (8/18/20) unsure why pylint is unable to recognize cython import...
```

Comment:
did you have a chance at all to take a look at why pylint can't recognize the cython import?